symphony-bdk-core/src/main/java/com/symphony/bdk/core/activity/command/CommandActivityInfo.java

```diff
@@ -0,0 +1,22 @@
+package com.symphony.bdk.core.activity.command;
+
+import com.symphony.bdk.core.activity.model.ActivityInfo;
+
+import lombok.Getter;
+import lombok.Setter;
+import lombok.experimental.Accessors;
+import org.apiguardian.api.API;
+
+/**
+ * {@link com.symphony.bdk.core.activity.command.CommandActivity} information/documentation model.
+ */
+@Getter
+@Setter
+@Accessors(fluent = true)
+@API(status = API.Status.EXPERIMENTAL)
+public class CommandActivityInfo extends ActivityInfo {
+
+  private String commandName;
+
+  private String summary;
```

Comment:
Difference between summary and description from parent class?

Reply:
the default slash command has this description: `description("Usage: " + (this.requiresBotMention ? "@BotMention " : "") + this.slashCommandName)`, that's why I thought it has the different use rather than a short description to be shown in the help command result.

Reply:
No problem to change that 👍

Reply:
I don't think we really use the description so far so we could change its default format (is it what you meant @symphony-thibault ?)